twilight-model/src/application/interaction/application_command/option.rs

```diff
@@ -360,6 +360,60 @@ pub enum CommandOptionValue {
     User(Id<UserMarker>),
 }
 
+impl From<Id<AttachmentMarker>> for CommandOptionValue {
+    fn from(value: Id<AttachmentMarker>) -> Self {
+        CommandOptionValue::Attachment(value)
+    }
+}
+
+impl From<bool> for CommandOptionValue {
+    fn from(value: bool) -> Self {
+        CommandOptionValue::Boolean(value)
+    }
+}
+
+impl From<Id<ChannelMarker>> for CommandOptionValue {
+    fn from(value: Id<ChannelMarker>) -> Self {
+        CommandOptionValue::Channel(value)
+    }
+}
+
+impl From<i64> for CommandOptionValue {
+    fn from(value: i64) -> Self {
+        CommandOptionValue::Integer(value)
+    }
+}
+
+impl From<Id<GenericMarker>> for CommandOptionValue {
+    fn from(value: Id<GenericMarker>) -> Self {
+        CommandOptionValue::Mentionable(value)
```

Comment:
It might be a good idea to exclude this one? Since things other than mentionables could be provided.

Reply:
Would you not always have the specific type in those cases, GenericMarker is not used that much.

Reply:
Maybe, maybe not. I would assert that there is room for users to be constructing IDs from eg a database value and their flow may not delineate between types. This could be argued to be the user's mistake, and so if they are using GenericMarker themselves so often as to make it, they "deserve" (for lack of a better word) to receive the error from Discord rather than from the library not supporting a single From implementation. I would leave whether that footgun is worth excluding the impl to you, but I did want to call it out as a consideration.

Reply:
I think that is a good enough reason to remove it, will do that later today.